.github/workflows/main.yml

```diff
@@ -91,3 +91,25 @@ jobs:
         RESULT_URL=$(cat response.txt |grep Location |awk '{print $2}')
         sleep 2
         http --ignore-stdin $RESULT_URL
+  wasm:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+
+      # Should not be needed anymore when https://github.com/NLESC-JCER/cpp2wasm/issues/1 is fixed
+      - name: Generate source code
+        uses: docker://nlesc/pandoc-tangle
```

Comment:
Can you fix the docker image version?

Reply:
Done in 93de4904b7aa01e63b6400674b288a24539dfea4